salt/states/blockdev.py

```diff
@@ -182,9 +174,7 @@ def formatted(name, fs_type="ext4", force=False, **kwargs):
         current_fs = _checkblk(name)
 
         if current_fs == fs_type:
-            ret["comment"] = ("{0} has been formatted " "with {1}").format(
-                name, fs_type
-            )
+            ret["comment"] = ("{} has been formatted " "with {}").format(name, fs_type)
```

Comment:
```suggestion
            ret["comment"] = "{} has been formatted with {}".format(name, fs_type)
```
:angry: 

there are other occurrences in the file...